contracts/wawaka/common/Attestation.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <string>
+
+#include "Value.h"
+
+namespace ww
+{
+namespace attestation
```

Comment:
As I complete support for CDI, which also has a notion of attestations, I think it would be good to disambiguate the different attestation objects. I'm wondering if the more specific `sgx_attestation` would be suitable here?

Reply:
possibly, but i would prefer not to generalize until we have more examples. we could put more functionality here from the contract-to-contract attestations. but i didn't find a good re-usable form. so... i would prefer to leave this until we have a more concrete proposal.

Reply:
Makes sense to me. There shouldn't be any namespace collisions right now either way, this is more of a readability/programmability concern anyhow.